identity/engines/pkcs11/engine_windows_test.go

```diff
@@ -0,0 +1,24 @@
+/*
+	Copyright 2019 NetFoundry, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package pkcs11
+
+import (
+	_ "net/url"
+)
+
+const pkcs11Lib = "V:/dev/tools/softhsm2/lib/softhsm2-x64.dll"
```

Comment:
ugh. need to convert this into an env var or something... any thoughts?

Reply:
If there is a default location use that as the const. Provide environment variable override. Ensure the logging tells someone which value is being used.

Reply:
make sure you have the latest PR to review. i addressed this